integration_test/scripts/run-store-scp-and-wait.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 # run_step scp_port scp_finish_port
-cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-storescp
+export PATH=/opt/apache-maven-3.6.3/bin:$PATH
+cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-storescp &&
 mvn exec:java -ntp -Dexec.mainClass=org.dcm4che3.tool.storescp.StoreSCP -Dexec.args='--bind STORESCP@'$1':'$2' --directory /workspace/integration_test/storescp-data --accept-unknown' &
 apt-get -qq update
 apt-get -qq install -y netcat-openbsd
 echo 'Waiting for connection on port '$3' to finish store-scp step'
-nc -l -p $3
+nc -l -p $3 &
```

Comment:
Shouldn't this not have an & to wait for the store scp to come up? Since & moves it to a background task?

Reply:
without & the process starts, but does not proceed to the next step in the script and we only see the logs that the process has started until the test stops by timeout. So we need &

Reply:
We're waiting for the port so we should be fine. Thanks

Reply:
Actually on second thought, this nc -l -p command is just waiting on that port. 
Us sending signals here is no longer required. It was previously a mechanism used to make steps complete successfully when we used the yaml file, but it's not needed now. This line can be deleted.